docs/UserGuide.md

```diff
@@ -215,34 +225,34 @@ Example usage:
 
 [Table of Contents](#table-of-contents)
 
-#### Delete some details: `delete [index of contact] [options]`
+#### Delete some details: `delete [index] [field prefix] [options]`
 
 You can also remove some information about a contact that you previously recorded.
 
 Format:
 
-* Delete phone: `delete [index of contact] /phone /index [index of phone]`
-* Delete email: `delete [index of contact] /email /index [index of email]`
-* Delete link: `delete [index of contact] /link /index [index of link]`
-* Delete graduation semester: `delete [index of contact] /grad`
-* Delete course: `delete [index of contact] /course /index [index of course]`
-* Delete specialisation: `delete [index of contact] /spec /index [index of specialisation]`
-* Delete priority: `delete [index of contact] /priority`
-* Delete tag: `delete [index of contact] /tag /index [index of tag]`
+* Delete **single-valued** field: `delete [index of contact] [field prefix]`
+* Delete an entry of a **multi-valued** field: `delete [index of contact] [field prefix] /index [index of entry]`
 
 Parameters:
 
-* `[index of contact]` is the index of the contact in the list
-* `/field` is the field of information to delete
-* `[index of field]` for a multi-valued field is the index of the entry in the contact's list of values
+* `[index of contact]` is the index of the contact in the list.
+* `[field prefix]` specifies the field of information to delete.
+* `[index of entry]` for a multi-valued field is the index of the element in the list representing that field.
+
+For **single-valued** fields, the `/index` prefix should not be used.
+
+For **multi-valued** fields, the `/index` prefix is optional and at most 1 index can be specified at a time. If not specified, the default index to delete is 1.
+
+<div markdown="block" class="alert alert-info">
 
-When you delete a single-valued field, the `/index` prefix should not be used.
+:information_source: To view a comprehensive list of fields and their prefixes, go to the start of the [Features section](#features).
 
-When you delete a multi-value field, the `/index` prefix is optional and at most 1 index can be specified at a time. If not specified, the default index to delete is 1.
+</div>
 
 Example usage:
 
-* `delete 1 /email /index 2`
+* `delete 1 /link /index 2`
 * `delete 5 /spec`
 * `delete 10 /priority`
```

Comment:
Did not mention in this in a previous PR, but should add an image of the app when an example input is executed.
You can add in this PR, or fix in a later PR.

Reply:
Will fix in this PR.

Reply:
> Will fix in this PR.

I realised that it's better to provide screenshots from the same OS for consistency. I'll ask @xenosf to help with this.

Reply:
@Singa-pirate Go ahead and merge this first, I will add a screenshot in a subsequent PR

Reply:
> @Singa-pirate Go ahead and merge this first, I will add a screenshot in a subsequent PR

Okay, thank you!